src/App.js

```diff
@@ -10,30 +10,24 @@ import { fetchMissions } from './redux/missions/missions';
 import './App.css';
 
 function App() {
-  // rockets
-  const rockets = useSelector((state) => state.rockets);
   const dispatch = useDispatch();
-
   useEffect(() => {
     dispatch(fetchRockets());
+    dispatch(fetchMissions());
   }, []);
 
+  const rockets = useSelector((state) => state.rockets);
   const reserveRockets = (id) => {
     dispatch(reserveRocket(id));
   };
```

Comment:
In my opinion, the first dispatch needs to be here, because the first fetch needs to be done on loading the _App_ not the components. However, Rocket-specific dispatches can stay in Rocket. It _can_ be done like this, but the button that will be using this function is in Rocket, and the beauty of Redux is that you don't have to pass functions up and down like in pure React.

Reply:
That sounds great. I will work on that surely